extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  const BRL = (price * 0.99 * 5.7);
```

Comment:
please use Caps like `BRL` only for some global static variables.
here, it's better to call something different, and because it a primitive value (number) better declate it wit let:
```js
let amount = ....

or 

let priceInBRL = ....
```